openssl-sys/build/main.rs

```diff
@@ -503,6 +503,7 @@ See rust-openssl README for more information:
             (7, _) => ('7', 'x'),
             (8, 0) => ('8', '0'),
             (8, 1) => ('8', '1'),
+            (8, 2) => ('8', '2'),
             (8, _) => ('8', 'x'),
```

Comment:
We already support 2.8.2 from this line here.

Reply:
Oh, indeed. I also noticed the `ctest` update fails on stable apparenlty. I'll see if I can find a fix.